pgml-cms/docs/api/sql-extension/pgml.rank.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Rank documents against a piece of text using the specified ranking model.
+---
+
+# pgml.rank()
+
+The `pgml.rank` function is used to rank text documents against some text. This function is primarly used as the last step in a search system where the results returned from the initial search are reranked before being used.
```

Comment:
```suggestion
The `pgml.rank()` function is used to rank text documents against some text. This function is primarily used as the last step in a search system where the results returned from the initial search are re-ranked before being used.
```

Reply:
https://www.reddit.com/r/HelixEditor/comments/10r5t56/spellcheck_in_helix/ No helix spellchecker? :O

Reply:
Haha no I need to add a spell checker LSP.